modules/fsi/src/kernels/AcousticInertia.C

```diff
@@ -0,0 +1,63 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "AcousticInertia.h"
+#include "SubProblem.h"
+#include "TimeIntegrator.h"
+#include "NonlinearSystemBase.h"
+
+registerMooseObject("FsiApp", AcousticInertia);
+
+InputParameters
+AcousticInertia::validParams()
+{
+  InputParameters params = TimeKernel::validParams();
+  params.addClassDescription("Calculates the residual for the inertial force "
+                             "which is the double time derivative of pressure.");
+  params.set<bool>("use_displaced_mesh") = true;
```

Comment:
Should you suppress use_displaced_mesh here and not let the user set it to false?